syntax = "proto2";

message ChangeCurrentSessionDayTemperatureMessage {
    required uint32 dayNumber = 2;
    required float temperature = 3;
}
